File: Interfaz/ProyectoFinal/src/app/Parametros/Entrada/AgregarModuloIn.ts
import { Modulo } from 'src/app/Components/Modulo/class/modulo';
import { TipoDeUsuario } from '../../Components/TipoDeUsuario/class/tipoDeUsuario';

export class AgregarModuloIn{
    
    private tipoDeUsuario: TipoDeUsuario;
    private modulo: Modulo;

    constructor()
    {
        this.tipoDeUsuario = new TipoDeUsuario();
        this.modulo = new Modulo();
    }

    public get TipoDeUsuario(): TipoDeUsuario {
        return this.tipoDeUsuario;
    }
    public set TipoDeUsuario(value: TipoDeUsuario) {
        this.tipoDeUsuario = value;
    }
    public get Modulo(): Modulo {
        return this.modulo;
    }
    public set Modulo(value: Modulo) {
        this.modulo = value;
    }

}

File: ProyectoFinal/ProyectoFinal/Parametros/Entrada/AgregarModuloIn.cs
using ProyectoFinal._2_Dominio.Entidades;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;

namespace ProyectoFinal.Parametros.Entrada
{
    public class AgregarModuloIn
    {
        public TipoDeUsuario TipoDeUsuario { get; set; }

        public Modulo Modulo { get; set; }
    }
}